app/templates/main/upload.html

```diff
@@ -40,33 +56,80 @@ <h1 class="govuk-heading-l">Get ready to submit your monitoring and evaluation d
                         "classes": notificationClass
                     }) }}
 
-                    <h2 class="govuk-heading-m">Upload your return for the April to September period</h2>
-                    <p class="govuk-body">You are uploading for: <b>{% for la in local_authorities %}<li class="govuk-body">{{ la }}</li>{% endfor %}</b> </p>
-                    <p class="govuk-body">You need to download the (LINK)[spreadsheet template]   so that you can complete the current return.</p>
-                    <br>
-                    <br>
+                    {% if local_authorities|length > 1 %}
+                        {% set localAuthorities %} {{ ", ".join(local_authorities) }} {% endset %}
+                    {% else %}
+                        {% set localAuthorities %} {{ " ".join(local_authorities) }} {% endset %}
+                    {% endif %}
```

Comment:
Bit confused about what this if statement is doing? Seems like the same code runs in both cases? Sorry if I missed something?

Reply:
there's a slight difference in the join method Gids

Reply:
If there's only 1 I don't think we need a join at all, can we refactor this?

Reply:
Seems like as it is, it will add an erroneous whitespace?

Reply:
I mentioned this and it's been refactored 👍